docs/pages/messages/invalid-env-prefix.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Invalid Environment Variable Prefix
+description: "invalid-env-prefix error page"
+---
+
+# Invalid Environment Variable Prefix
+
+When declaring environment variables in your `turbo.json`, you cannot prefix them with `$`. This
+was an old syntax for declaring a dependency on an environment variable that is no longer used.
```

Comment:
"This syntax was deprecated in 1.5. You can migrate to the `env` and `globalEnv` keys using npx `@turbo/codemod migrate-env-var-dependencies`. [Check out the codemod's documentation](https://turbo.build/repo/docs/reference/codemods#migrate-env-var-dependencies) for more.